src/api/spec/models/staging/workflow_spec.rb

```diff
@@ -79,16 +77,21 @@
       it { expect(subject).to be_empty }
     end
 
-    context 'with requests and some are in staging projects and some not' do
-      let!(:bs_request_2) do
+    context 'with request in state review whether they are in staging projects or not' do
+      let(:bs_request_2) do
         create(:bs_request_with_submit_action,
-               target_package: target_package,
-               source_package: source_package)
+               state: :review,
+               review_by_group: group, # FIXME: force to create a review for the group, because skip_sanitize is avoiding it
+               target_project: project.name,
```

Comment:
please don't revert my changes - aka don't pass .name but the object and don't repeat the projects, but let the bsrequest factory handle that.

I'm not sure what you mean with the FIXME, but I'm happy to have a look